cryptography.py

```diff
@@ -1,12 +1,43 @@
 """
 cryptography.py
-Author: <your name here>
-Credit: <list sources used, if any>
+Author: Jackson Lake
+Credit: 
 
 Assignment:
 
 Write and submit a program that encrypts and decrypts user data.
 
 See the detailed requirements at https://github.com/HHS-IntroProgramming/Cryptography/blob/master/README.md
 """
+#convert each one to a number that represents it, operate on that number, then convert back to a letter.
+
 associations = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789 .,:;'\"/\\<>(){}[]-=_+?!"
+command = "z"
+
+
+while command != "q":
+    command = input("Enter e to encrypt, d to decrypt, or q to quit: ")
+    if command == "e" or command == "d":
+        message = input("Message: ")
+        key = input("Key: ")
+        if len(message) > len(key):
+            key = len(message)*key
+        nums=[]
+        if command == "e":
+            for i in range(len(message)): 
+                thing = associations.find(message[i]) + associations.find(key[i])
+                if thing >= 86:
+                    associations = associations*2
+                print(associations[thing], end = '')
+            print()
+        if command == "d":
```

Comment:
```python
elif command == "d":  # ?
```
If `command` is equal to "e", it cannot also be equal to "d".

Reply:
Thanks for the suggestion? I will try to fix it. Thanks for checking out my projects. By the way, I have another project coming where I am making a simple game. Feel free to check that one out and add comments or actual code.